repo/packages/M/marathon/0/marathon.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
Had a quick look at the mustache repository and it looks like they use `*.mustache`: https://github.com/mustache/mustache/tree/master/examples